assets/js/base/components/payment-methods/style.scss

```diff
@@ -150,6 +150,17 @@
 
 .wc-blocks-credit-card-images {
 	padding-top: $gap-small;
+	display: flex;
+
+	.stripe-icon {
+		height: 18px;
+		width: auto;
+		margin-right: $gap-small;
+
+		&:last-child {
+			margin-right: 0;
+		}
+	}
```

Comment:
The name of the class is too specific to payment method. Anything in our components should be generic so that it can be used by other payment methods.  So instead of `.stripe-icon` maybe call it `.wc-blocks-credit-cart-icon` (and then if payment methods want to get the styles for free, they use that class)?

Reply:
I went with the same classes that are used by that the gateway to ensure compatibility but I can change that.

Reply:
> I went with the same classes that are used by that the gateway

That makes sense, but the reason I'm asking for the change is:

- we're intentionally trying to keep the gateway specific stuff out of the blocks plugin and limited to the payment method builds. This makes it easier to do an extraction when we move the payment method to the relevant extension.
- we want to avoid gateway specific styles and incorporate styles that _could_ be implemented by any gateway to align with the overall style of the blocks. So using more generic naming helps signal that.
- If a gateway/payment method needs to add it's own styling then that css should be built with the payment method file (but so far I haven't seen anything that suggests that needs to happen).

Reply:
This feels logical to me.